Chapter07/bookmarks/images/models.py

```diff
@@ -18,7 +18,8 @@ class Image(models.Model):
     users_like = models.ManyToManyField(settings.AUTH_USER_MODEL,
                                         related_name='images_liked',
                                         blank=True)
-    total_likes = models.PositiveIntegerField(default=0)
+    total_likes = models.PositiveIntegerField(db_index=True, 
```

Comment:
this is covered by class Meta:
        indexes = [
            models.Index(fields=['-created']),
            models.Index(fields=['-total_likes']),

Reply:
chapter 07 (Word doc), page 32 has it as:
total_likes = models.PositiveIntegerField(db_index=True,
                                              default=0)
that was the reason I altered it - 
I assumed (possibly mistakenly) the text was correct?

Reply:
@auvipy is right, @kstingel this needs to be fixed the Word doc. The index for `total_likes`is included in the `indexes` of the `Meta` class. Same issue with the `target_id` field of the `Action` model in the `actions` app, which also has to be removed in the Word doc examples.